dev-packages/e2e-tests/test-applications/aws-serverless-esm/src/lambda-function.mjs

```diff
@@ -0,0 +1,26 @@
+import * as http from 'node:http';
+import * as Sentry from '@sentry/aws-serverless';
+
+const handler = Sentry.wrapHandler(async () => {
+  await new Promise(resolve => {
+    const req = http.request(
```

Comment:
took me ages to find out why the previously used `http.get` call didn't produce a span. Turns out it's an Otel ESM bug. Reported it to Otel here: https://github.com/open-telemetry/opentelemetry-js/issues/4857

Reply:
nice find